gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -342,6 +350,16 @@ def add_arguments(self, parser):
               'pass over all variants. Additionally, this flag will resolve '
               'conflicts for all headers as if `allow_incompatible_types` was '
               'true.'))
+    parser.add_argument(
+        '--run_with_garbage_collection',
```

Comment:
Why not always do it?

Reply:
Yes we should always do it. I keep it for the first version just in case when something unexpected happens, we can still run previous annotation version correctly (like use the previous image, no sharding at all, etc.). Does it make sense to you?